digitalocean/datasource_digitalocean_container_registry.go

```diff
@@ -0,0 +1,78 @@
+package digitalocean
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/digitalocean/godo"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+const RegistryHostname = "registry.digitalocean.com"
+
+func dataSourceDigitalOceanContainerRegistry() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceDigitalOceanContainerRegistryRead,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				Description:  "name of the container registry",
+				ValidateFunc: validation.NoZeroValues,
+			},
+			//TODO: Need better name?
+			"write": {
```

Comment:
Maybe `read_write`? That matches the actual query parameter used and makes the context a bit more clear.

https://developers.digitalocean.com/documentation/v2/#get-docker-credentials-for-container-registry

Reply:
I suggested `write` because I was initially confused by the real parameter name. I think others might be as well. Don't have strong feelings though.

Reply:
@andrewsomething any further preference here?

Reply:
No strong preference. Mostly made a suggestion because of the `TODO` comment.

Reply:
@Zelgius I prefer `write` because `read_write` isn't as clear as what it does since read is true by default. Feel free to make the call and remove the `TODO`.